frontend/src/components/Timer.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+<div>
+	<div class="wrapper">
+		<h1>Timer</h1>
+		<br>
+		<h1><span id="seconds">00</span>:<span id="tens">00</span></h1>
+		<br> <br>
+		<div class="buttons">
+			<button id="button-start">Start</button>
+			<button id="button-stop">Stop</button>
+			<button id="button-reset">Reset</button>
+		</div>	
+</div> 
+</div>
+</template>
+
+<script>
+	window.onload = function () {
```

Comment:
I just put it as a proof of concept! haha. I only started to pull request to compare the branches anyway.